users/contants.py

```diff
@@ -14,3 +14,34 @@
     SALES_MANAGER: "Sales Manager",
     SYSTEM_ADMIN: "System Admin",
 }
+# directory containing css file
+LOGIN_CSS_DIRECTORY = "/login_register/style.css"
```

Comment:
So, basically, we would create a single css file for a single app and that file would contain all css data? Or did i misunderstand this

Reply:
No, We should create multiple CSS files w.r.t to the separation of concern in mind. My comment talks about keeping the same directory for CSS files and instead of defining constants here and passing them in context, let's directly load the one or more css within the HTML files.

Reply:
Oh Ok, i get it now. Thanks for clarifiation